app/src/lib/stores/app-store.ts

```diff
@@ -471,7 +474,11 @@ export class AppStore extends TypedBaseStore<IAppState> {
         return null
       }
 
-      return { type: SelectionType.CloningRepository, repository, progress }
+      return {
```

Comment:
It's unclear why we are returning an object of this shape. A bit of documentation  describing `PossibleSelections` and it's variants would go a long way in helping me understand it.

Reply:
this was simply a formatting change by `prettier`. @iAmWillShepherd do you want to file a new issue for some docs around this?